rpc/client/localclient.go

```diff
@@ -27,6 +33,7 @@ powerful control during testing, you probably want the "client/mock" package.
 */
 type Local struct {
```

Comment:
don't quite understand why we need Local client

Reply:
I would think for testing and other in-process use to avoid HTTP connections.